c2corg_ui/static/js/viewdetails.js

```diff
@@ -133,6 +142,27 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: this.compile_(template)(this.scope_)});
 };
 
+/**
+ * @param {Object} doc
+ * @private
+ */
+app.ViewDetailsController.prototype.removeAssociation_ = function(doc) {
+  var associations;
+
+  if (doc['type'] === 'outings') {
+    associations = this.scope_['document']['associations']['recent_outings']['outings'];
+  } else {
+    associations = this.scope_['document']['associations'][doc['type']];
+  }
+  if (associations) {
+    for (var i = 0; i < associations.length; i++) {
```

Comment:
Good catch! I think this function should be replaced by the one in the ``documentService`` (perhaps with the test about ``recent_outings`` and maybe ``all_routes``). Since such a change might require intensive testing to make sure nothing is broke, I would prefer it is done in another PR. I will create an issue.